.gitignore

```diff
@@ -1,12 +1,5 @@
-/.sass-cache/
-/export/
+/build/
 /node_modules/
-/pattern-library-*.tar.gz
-/public/
```

Comment:
Presume `/public` removed as this is not exposed to the host?

Reply:
Moved inside `build`.